src/vaadin-progress-bar.ts

```diff
@@ -2,6 +2,17 @@ import { customElement } from 'lit-element';
 import { ProgressBarBase } from './vaadin-progress-bar-base';
 import { ProgressMixin } from './vaadin-progress-mixin';
 
+/**
+ * `<vaadin-progress-bar>` is a Web Component for progress bars.
+ * Content can be any HTML, including nested `<vaadin-progress-bar`.
+ *
+ * @csspart part="bar" - Progress-bar's background
+ * @csspart part="value" - Progress-bar's foreground
+ *
+ * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin/wiki)
```

Comment:
Please remove `ThemableMixin`, we don't mention it in the docs for next generation components.

Reply:
done